### PR TITLE
docs: update examples in `stats/base/dists/uniform`

### LINKED ISSUE #1648: [RFC]: improve README examples of `stats/base/dists/uniform` namespace
### Description

This RFC proposes to improve the [README examples](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/base/dists/uniform#examples) of the `stats/base/dists/uniform` namespace package. Currently, namespace package READMEs only include minimal examples which are not particularly informative and do not showcase namespace functionality.

Historically, namespace examples were not given much effort due to the rapidly changing nature of the project; however, now that development and organization has settled, it would be good to revisit these packages and add proper examples to better communicate to users how to use namespace contents.

For those wanting to contribute to this issue, you can use the following steps:

- Browse through the [contents](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/base/dists/uniform) of the `stats/base/dists/uniform` namespace and identify interesting functions and packages to showcase.
- Update the example code block in the "Examples" section of the README for the `stats/base/dists/uniform` namespace and open a PR.
- When updating the README, examples should follow existing project conventions (e.g., ES5).
- Examples may `require` functionality from other packages to help make examples more interesting.

### Related Issues

None.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by headlessNode**
Resolves # .

## Description

> What is the purpose of this pull request?

This pull request:

-   Update the example code block in the "Examples" section of the README for the `stats/base/dists/uniform` namespace

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #1648 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/stats/base/dists/uniform/README.md [hunk 1]
```diff
@@ -108,10 +108,39 @@ var y = dist.cdf( 2.5 );
...
+
+function calculatePdfs(grade) {
+    return uniform.pdf(grade, lowerBound, upperBound).toFixed(2);
+}
+
+var cdfsGrades = grades.map(calculateCdfs);
+// returns '[ '0.85', '0.70', '0.92', '0.65', '0.78', '0.88', '0.95', '0.60', '0.72', '0.90' ]'
+
+var pdfsGrades = grades.map(calculatePdfs);
+// returns [ '0.01', '0.01', '0.01', '0.01', '0.01', '0.01', '0.01', '0.01', '0.01', '0.01' ]
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/uniform/README.md]: @headlessNode I recognize that you've shown how to invoke the various methods, but I am not following why a reader should be interested in this particular example. What would be the point of, e.g., calculating the entropy? Why would that be useful to calculate? Same thing for the PDFs? What information does that provide the reader?

I think for the purposes of adding namespace examples, we'd be interested in something which tells a better story.

Comment (headlessNode) [reply lib/node_modules/@stdlib/stats/base/dists/uniform/README.md]: Thank you for your feedback! Now that you've highlighted the issue, I can see that the calculation of Entropy, CDF, and PDF may not be relevant or necessary for this example scenario. I appreciate you bringing this to my attention. I'll revise the examples.

Comment (headlessNode) [reply lib/node_modules/@stdlib/stats/base/dists/uniform/README.md]: @kgryte What do you think about Analyzing daily Temperature data? The study of the temperature distribution to understand the trends in temperature readings would make more sense?

Comment (kgryte) [reply lib/node_modules/@stdlib/stats/base/dists/uniform/README.md]: Sure, that could work. It really just depends on how you plan to use/showcase the namespace APIs.

Comment (headlessNode) [reply lib/node_modules/@stdlib/stats/base/dists/uniform/README.md]: @kgryte I've developed an example scenario where we analyze the effectiveness of a  HVAC system in a building. 

I think that making it even more engaging would involve the use of other methods or external resources and make it out of scope for our current objectives

Before proceeding with these changes, I wanted to reach out to you for feedback. Do you think this approach effectively addresses the concerns raised in your feedback? Are there any additional suggestions or modifications you would recommend?

```
/*
* Let's consider an example where we're analyzing the effectiveness of a new energy-efficient HVAC system in a building.
* The system aims to maintain optimal indoor temperatures while minimizing energy consumption. 
* We can calculate the mean, variance, and standard deviation of the temperature readings to assess the system's performance in maintaining consistent and comfortable temperatures. 
* We can use the entropy to evaluate the system's efficiency in adapting to changing environmental conditions and optimizing energy usage.
* The cumulative distribution function (CDF) and probability density function (PDF) can help us understand the distribution of temperature values within the building.
*/

// Assume we have a list of daily temperature readings in the building
var temperatures = [25, 28, 26, 30, 27, 29, 31, 32, 24, 26];

var lowerBound = 20; // Minimum possible temperature
var upperBound = 35; // Maximum possible temperature

// Represents the average temperature observed in the building
var meanTemperature = uniform.mean(lowerBound, upperBound).toFixed(2);
// returns 27.50

// Measures how much the temperature readings deviate from the mean temperature.
var varianceTemperatures = uniform.variance(lowerBound, upperBound).toFixed(2);
// returns 18.75

// Measures the amount of variation of a set of values, providing insight into the consistency or stability of indoor temperatures.
var stdDevTemperatures = uniform.stdev(lowerBound, upperBound).toFixed(2);
// returns 4.33

// Measures the uncertainty or randomness in indoor temperature patterns
var entropyTemperatures = uniform.entropy(lowerBound, upperBound).toFixed(2);
// returns 2.71

// Provides the probability that the temperature is less than or equal to a given value, aiding in understanding the likelihood of specific temperature ranges occurring within the building.
function calculateCdfs(temperature) {
    return uniform.cdf(temperature, lowerBound, upperBound).toFixed(2);
}

// Provides the probability of a temperature falling within a particular range, aiding in identifying temperature patterns within the building.
function calculatePdfs(temperature) {
    return uniform.pdf(temperature, lowerBound, upperBound).toFixed(2);
}

var cdfsTemperatures = temperatures.map(calculateCdfs);
// returns '['0.33', '0.53', '0.40', '0.67', '0.47', '0.60', '0.73', '0.80', '0.27', '0.40']'

var pdfsTemperatures = temperatures.map(calculatePdfs);
// '['0.07', '0.07', '0.07', '0.07', '0.07', '0.07', '0.07', '0.07', '0.07', '0.07']'

```

Comment (headlessNode) [reply lib/node_modules/@stdlib/stats/base/dists/uniform/README.md]: @kgryte I've added the above changes, developed an engaging story and mentioned what each result means in that context. Please review. Thanks.

**Review by Planeshifter — APPROVED** [review]

**Pull request merged**